src/test/integration/resources/application.properties

```diff
@@ -41,4 +45,10 @@ nocNotification.template.respondent.id=${NOC_NOTIFICATION_TEMPLATE_RESPONDENT_ID
 nocNotification.template.claimant.id=${NOC_NOTIFICATION_TEMPLATE_CLAIMANT_ID:test}
 nocNotification.template.previousrespondentsolicitor.id=${NOC_NOTIFICATION_TEMPLATE_OLD_REP_ID:test}
 nocNotification.template.newrespondentsolicitor.id=${NOC_NOTIFICATION_TEMPLATE_NEW_REP_ID:test}
-nocNotification.template.tribunal.id=${NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID:test}
\ No newline at end of file
+nocNotification.template.tribunal.id=${NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID:test}
+
+
+tse.admin.record-a-decision.claimant.template.id=4d0fdc95-8f48-4a67-9713-3c8474044ad2
```

Comment:
all properties should follow ${NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID:test} format. env variable plus dummy

Reply:
agreed, resolved.